kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/cmd/BaseCmdKubeClient.java

```diff
@@ -91,6 +91,18 @@ public K deleteByName(String resourceType, String resourceName) {
         return (K) this;
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public K deleteAllByResource(String resourceType) {
+        try {
+            Exec.exec(namespacedCommand(DELETE, resourceType, "--all"));
+        }
+        catch (Exception e) {
+            LOGGER.warn(e.getMessage());
```

Comment:
I question why swallowing the exception is the right thing to do.
Also - **always** pass the exception to log methods.  Without it you have no context to understand the problem.